app/app.go

```diff
@@ -922,10 +1028,30 @@ func NewEvmos(
 	return app
 }
 
+// CheckTx will check the transaction with the provided checkTxHandler. We override the default
+// handler so that we can verify bid transactions before they are inserted into the mempool.
+// With the POB CheckTx, we can verify the bid transaction and all of the bundled transactions
```

Comment:
Please, do not use acronyms in a file without specifying their meaning at least once.